x-pack/plugins/observability/server/routes/slo/route.ts

```diff
@@ -372,6 +373,25 @@ const findSLORoute = createObservabilityServerRoute({
   },
 });
 
+const findSLOGroupsRoute = createObservabilityServerRoute({
+  endpoint: 'GET /internal/api/observability/slos/_groups 2023-10-31',
```

Comment:
You don't need the date versioning if it's an internal route.

Reply:
ok will remove it

Reply:
Done